contracts/strategies/DeltaStrategy.sol

```diff
@@ -50,122 +72,306 @@ contract DeltaStrategy is Ownable {
     // vault.startWithdrawPeriod
   }
 
+  ///////////////////
+  // VAULT ACTIONS //
+  ///////////////////
+
+  function setBoard(uint boardId) external onlyVault {
+    Board memory board = getBoard(boardId);
+    require(isValidBoard(board), "invalid board");
+    activeExpiry = board.expiry;
+  }
+
+  function returnFundsAndClearStrikes() external onlyVault {
+    ExchangeRateParams memory exchangeParams = getExchangeParams();
+    uint quoteBal = quoteAsset.balanceOf(address(this));
+    uint baseBal = baseAsset.balanceOf(address(this));
```

Comment:
better move this line into the if block below, so we don't need to call it when `isBaseCollat() = false`